src/pages/trainer/training/edit/components/toolbar/icons/italicIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+export const ItalicIcon = () => {
+  return (
+    <span className="glyphicon glyphicon-italic"/>
+  );
+};
```

Comment:
Not sure which approach to choose, once you read it in the toolbar presentational it's readable and nice, but when I see the detail of each component taking into account that are very similar that's doesn't smell well. Not sure, what's your opinion?